turbinia/workers/file_system_timeline.py

```diff
@@ -81,9 +82,9 @@ def run(self, evidence, result):
       status = 'Generated file system timeline containing [{0:d}] entries'.format(
           number_of_entries)
       result.close(self, success=True, status=status)
-    except dfvfs_errors.ScannerError as exception:
-      result.log('Error generating bodyfile {0!s}'.format(exception))
+    except (dfvfs_errors.ScannerError, dfvfs_errors.BackEndError) as exception:
       status = 'Unable to generate bodyfile using provided evidence data.'
```

Comment:
Optional, but should we update this to indicate that there was an error?

Reply:
Done